packages/licenses/src/index.ts

```diff
@@ -0,0 +1 @@
+export { LicensePackage, licences } from './licenses'
```

Comment:
use a verb for functions. Maybe `licenses=>findDependencyLicenses`

Reply:
Rename the function